samcli/commands/list/cli_common/list_common_context.py

```diff
@@ -0,0 +1,33 @@
+"""
+Common context class to inherit from for sam list sub-commands
+"""
+import boto3
```

Comment:
let's move all boto3 imports within wherever they are required. It can incur upto 200ms extra load time.